nexmark-flink/src/main/java/com/github/nexmark/flink/FlinkNexmarkOptions.java

```diff
@@ -37,9 +37,9 @@
 	public static final ConfigOption<Duration> METRIC_MONITOR_DURATION = ConfigOptions
 		.key("nexmark.metric.monitor.duration")
 		.durationType()
-		.defaultValue(Duration.ofDays(1))
-		.withDescription("How long to monitor the metrics, default 1day, " +
-			"large value, monitor until job is finished.");
+		.defaultValue(Duration.ofMillis(Long.MAX_VALUE))
```

Comment:
I still prefer to use `3min` as default value here, and set it to MAX when in `events.num` mode. It affect users to have to set this option manually, otherwise, job is never stopped in tps mode (a regression).

Reply:
Actually, we already recommend user to set this in the configuration file. So I don't think this is a regression.

If we set this default value to `3min`, but it is the MAX in events.num mode.
How to explain to users? Because in my intuition, monintor duration can work with events.num mode, they are not in complete conflict.

So I think keep consistency is more clear.

Reply:
Consider user config this monitor duration in the config in events.num mode, but it doesn't work, I think this is not good.